src/libraries/Common/src/Interop/Windows/Advapi32/Interop.RegEnumValue.cs

```diff
@@ -13,8 +13,8 @@ internal static partial class Interop
 {
     internal static partial class Advapi32
     {
-        [DllImport(Libraries.Advapi32, CharSet = CharSet.Unicode, BestFitMapping = false, EntryPoint = "RegEnumValueW", ExactSpelling = true)]
-        internal static extern int RegEnumValue(
+        [GeneratedDllImport(Libraries.Advapi32, EntryPoint = "RegEnumValueW", CharSet = CharSet.Unicode, ExactSpelling = true)]
```

Comment:
are we assuming BestFitMapping = true is fine in these?

Reply:
Yeah, these are all using `CharSet.Unicode`, so `BestFitMapping` shouldn't matter.

Reply:
It seems like there are several variations of DllImport/GeneratedDllImport properties that are not meaningful or invalid but just ignored. I wonder whether it would make sense for the generator to flag/reject those so the ycan be cleaned up during conversion? I guess it is a minor thing.

Reply:
Certainly something we should keep in mind. We've been trying to remove support for sometimes-non-meaningful things (like `BestFitMapping`) from `GeneratedDllImport`, so hopefully it will be in a much better replace than `DllImport` in terms of weird properties and combinations.